pages/LearnerDashboard.html

```diff
@@ -0,0 +1,41 @@
+<section id="main_content" class="inner">
+  <h2>
+    <a id="Learner Dashboard" class="anchor"
+       href="#Learner_Dashboard" aria-hidden="true">
+      <span class="octicon octicon-link"></span>
+    </a>
+    Learner Dashboard
+  </h2>
+
+  <p>
+    Oppia's mission from the very start has been - "To help anyone learn anything they want in an effective and enjoyable way.". The "Learner Dashboard" has been designed and developed keeping this goal in mind. If you're a new user and you have signed in for the first time, here's how the learner dashboard would look -
```

Comment:
Oppia's mission from the very start has been "to help anyone learn anything they want in an effective and enjoyable way."